dask/dataframe/tests/test_groupby.py

```diff
@@ -3632,3 +3633,48 @@ def test_groupby_numeric_only_true(func):
         ddf_result = getattr(ddf.groupby("A"), func)(numeric_only=True)
         pdf_result = getattr(df.groupby("A"), func)(numeric_only=True)
         assert_eq(ddf_result, pdf_result)
+
+
+@pytest.mark.parametrize("func", ["cumsum", "cumprod"])
+def test_groupby_numeric_only_false(func):
+    df = pd.DataFrame(
+        {
+            "int": [1, 2, 3, 4, 5, 6, 7, 8],
+            "float": [1.0, 2.0, 3.0, 4.0, np.nan, 6.0, 7.0, 8.0],
+            "dt": [pd.NaT] + [datetime(2010, i, 1) for i in range(1, 8)],
+            "A": 1,
+        }
+    )
+    ddf = dd.from_pandas(df, npartitions=2)
+
+    if PANDAS_GT_200:
+        ctx = pytest.raises(TypeError, match="does not support")
+
+        with ctx:
+            getattr(ddf.groupby("A"), func)(numeric_only=False)
```

Comment:
Is there a reason to not parametrize with `numeric_only`?

Reply:
That would make it a big ugly (imo) in the else branch